test/java/org/opendatakit/common/cli/ParamTest.java

```diff
@@ -0,0 +1,30 @@
+package org.opendatakit.common.cli;
+
+import org.junit.Test;
+import org.opendatakit.briefcase.reused.BriefcaseException;
+
+import java.time.LocalDate;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class ParamTest {
+
+  @Test(expected = BriefcaseException.class)
+  public void throwBriefcaseExceptionForInvalidDateFormat() {
+      Param<LocalDate> localDate = Param.localDate("start", "export_start_date", "Export start date");
+      localDate.map("2018.01.20");
+  }
+
+  @Test
+  public void acceptsIso8601Dates () {
+      Param<LocalDate> localDate = Param.localDate("start", "export_start_date", "Export start date");
+      assertThat(localDate.map("2018-01-20").isEqual(LocalDate.of(2018, 01,20)), is(true));
```

Comment:
These assertions are not leveraging Hamcrest matchers correctly. To see that, change the expected local date to produce a test failure and see the error message:

```xunit
java.lang.AssertionError: 
Expected: is <true>
     but: was <false>
Expected :is <true>
     
Actual   :<false>
```

Now compare it when you change the assertion to this:

```java
assertThat(localDate.map("2018-01-20"), is(LocalDate.of(2018, 2,20)));
```

```
java.lang.AssertionError: 
Expected: is <2018-02-20>
     but: was <2018-01-20>
Expected :is <2018-02-20>
     
Actual   :<2018-01-20>
```

That failure message is much more useful.